channel/allocation.go

```diff
@@ -601,7 +600,7 @@ func AssetsAssertEqual(a []Asset, b []Asset) error {
 	}
 
 	for i, asset := range a {
-		if ok, err := equalMarshaling(asset, b[i]); err != nil {
+		if ok, err := perunio.EqualBinary(asset, b[i]); err != nil {
```

Comment:
We are currently working on a #279  that introduces the Asset type with an Equal method. This should be then used here, depending on which we merge first.

Reply:
https://github.com/hyperledger-labs/go-perun/pull/272#discussion_r763705784